src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs

```diff
@@ -22,6 +22,26 @@ public abstract partial class Workspace
         private readonly Dictionary<DocumentId, TextTracker> _textTrackers = new Dictionary<DocumentId, TextTracker>();
 
         /// <summary>
+        /// Unloading projects takes a serialization lock, which can cause Shared Project
```

Comment:
Can you name which function that lock is being taken in here? With a cref if possible? 😉